src/content/docs/validator/amplifier/verifier-onboarding.mdx

```diff
@@ -450,17 +450,27 @@ Prior to running the `ampd` daemon, you will need to set up your wallet with dev
 
 1. Bond your verifier:
 
-   ```bash
-   ampd bond-verifier amplifier 100000000000 uaxl
-   ```
-
-   Bonded verifiers will stay bonded until they unbond or are removed by governance. The bond for testnet is 100000000000uaxl (100k AXL). The bond for mainnet is 50000000000uaxl (50k AXL).
+   <tabs>
+   <tab-item title="devnet-amplifier">
+      ```bash
+      ampd bond-verifier validators 100000000000 uamplifier
```

Comment:
isn't the testnet bond 100 uamplifier?

Reply:
double checked it's `uaxl` for testnet `uamplifier` is only for devnet